AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1088,7 +1088,13 @@ - (void)__layout
   // with negative sizes after applying margins, which will cause
   // measureWithSizeRange: on subnodes to assert.
   if (!CGRectEqualToRect(bounds, CGRectZero)) {
+    // Handle placeholder layer creation in case the size of the node changed after the initial placeholder layer
+    // was created
```

Comment:
@maicki In a followup can you clarify this comment?  Why is it that the placeholder needs to be re-created after the node size changes?  Basically just describe what operation / action takes place that results in the need to call this method again.

I think the condition would be simplified to put _shouldHave... up in the if() at the top that checks for zero bounds.  Then we can unconditionally call _setupPlaceholder... and do the .frame = set.